src/tools/cleaner.js

```diff
@@ -46,18 +54,15 @@ const recurseReplaceBank = (obj, bank) => {
   return recurseReplace(obj, replacer);
 };
 
-// Takes a raw app and bundle and composes a bank of {{key}}->val
-const createBundleBank = (appRaw, event) => {
-  appRaw = appRaw || {};
-  event = event || {};
+const finalizeBundle = pipe(
+  pick(Object.keys(DEFAULT_BUNDLE)),
+  defaults(DEFAULT_BUNDLE)
+);
 
-  const bundle = event.bundle || {};
+// Takes a raw app and bundle and composes a bank of {{key}}->val
+const createBundleBank = (appRaw, event = {}) => {
   const bank = {
-    bundle: _.merge(
-      {},
-      { authData: bundle.authData || {} },
-      { inputData: bundle.inputData || {} }
-    ),
+    bundle: finalizeBundle(event.bundle),
     process: {
       env: _.extend({}, process.env || {})
```

Comment:
This kind of scares me since any integration can access `process.env`. Are we sure there is no sensitive data in `process.env`. Should we be filtering out any values?

Reply:
discussed in slack, this is fine